README.md

```diff
@@ -1,32 +1,68 @@
 # Tanzu Core CLI
 
 :warning: NOTE: This repository is still under early development. We plan to
-make releases available for evaluation in the second quarter of 2023.  Please
+make releases available for evaluation in the second quarter of 2023. Please
 watch this page for further updates.
 
 ## Overview
 
-The Tanzu Core CLI project provides the core functionality of the Tanzu CLI.
+VVV: diagram maybe?
```

Comment:
Let's remove this VVV comment.  I think we can start without a diagram for the moment

Reply:
Addresses as part of https://github.com/vmware-tanzu/tanzu-cli/pull/178/